### PR TITLE
flake8 passes

### PR DESCRIPTION
**Pull request by maxim-lian**
I was still getting stickler errors for code I hadn't changed. Flake8 should now pass

### CODE REVIEW

**Comment by shoyer** [comment]
Huh. I wonder how this got past our new CI tests for flake8.

**Comment by maxim-lian** [comment]
It's v weird. If an ignore parameter is **added**, it generates the errors. This one on master:

```
# flake8  xarray                                                                                                         
-> passes

# flake8 --ignore=I002 xarray
xarray/plot/plot.py:762:47: E226 missing whitespace around arithmetic operator
xarray/plot/plot.py:764:47: E226 missing whitespace around arithmetic operator
xarray/backends/rasterio_.py:193:34: E226 missing whitespace around arithmetic operator
xarray/backends/rasterio_.py:193:52: E226 missing whitespace around arithmetic operator
xarray/backends/rasterio_.py:194:33: E226 missing whitespace around arithmetic operator
xarray/backends/rasterio_.py:194:52: E226 missing whitespace around arithmetic operator
xarray/tests/test_indexing.py:117:28: E241 multiple spaces after ','
xarray/tests/test_indexing.py:122:28: E241 multiple spaces after ','
xarray/tests/test_indexing.py:122:35: E241 multiple spaces after ','
xarray/tests/test_indexing.py:122:42: E241 multiple spaces after ','
xarray/tests/test_indexing.py:129:28: E241 multiple spaces after ','
xarray/tests/test_indexing.py:132:28: E241 multiple spaces after ','
xarray/tests/test_indexing.py:135:28: E241 multiple spaces after ','
xarray/tests/test_indexing.py:135:35: E241 multiple spaces after ','
xarray/tests/test_indexing.py:135:42: E241 multiple spaces after ','
xarray/tests/test_backends.py:2217:28: E226 missing whitespace around arithmetic operator
xarray/tests/test_backends.py:2217:31: E226 missing whitespace around arithmetic operator
xarray/tests/test_backends.py:2234:55: E226 missing whitespace around arithmetic operator
xarray/tests/test_backends.py:2235:66: E226 missing whitespace around arithmetic operator
xarray/tests/test_backends.py:2236:65: E226 missing whitespace around arithmetic operator
xarray/tests/test_backends.py:2237:30: E126 continuation line over-indented for hanging indent
xarray/tests/test_backends.py:2319:36: E226 missing whitespace around arithmetic operator
xarray/tests/test_backends.py:2319:39: E226 missing whitespace around arithmetic operator
xarray/tests/test_duck_array_ops.py:21:31: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:21:37: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:21:43: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:22:31: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:22:37: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:22:43: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:23:30: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:23:36: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:23:42: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:25:31: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:25:37: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:25:43: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:26:31: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:26:37: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:26:43: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:27:31: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:27:37: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:27:43: E241 multiple spaces after ','
xarray/tests/test_duck_array_ops.py:290:42: E226 missing whitespace around arithmetic operator
xarray/tests/test_duck_array_ops.py:295:34: E226 missing whitespace around arithmetic operator
```

Identical results if `--isolated` is passed

**Comment by maxim-lian** [comment]
Ah - problem solved - it's that there are some [default ignores](http://flake8.pycqa.org/en/3.1.1/user/options.html#cmdoption-flake8--ignore). So on adding an ignore, those are then picked up again.

File: setup.cfg [hunk 1]
```diff
@@ -6,3 +6,6 @@ python_files=test_*.py
 
 [flake8]
 max-line-length=79
+ignore=I002
+exclude=
+  doc/
```
Comment (shoyer) [inline setup.cfg]: Can you add this configuration to the `.stickler.yml` file, too? That will help keep things in sync.

See https://stickler-ci.com/docs#python

Comment (maxim-lian) [reply setup.cfg]: Good idea, done 
(though ideally it should read from standard configs...)

**Comment by shoyer** [comment]
Thanks @maxim-lian

File: setup.cfg (hunk 1 above)
Comment (shoyer) [reply setup.cfg]: It looks like this didn't actually work to ignore the doc directories -- see all the Stickler messages in https://github.com/pydata/xarray/pull/1930

**Pull request merged**